### PR TITLE
Speed-ups and timing

### PR DESCRIPTION
**Pull request by jwoillez**
Dear @romanz,
I'm using `amodem` on a little raspberry pi, and needed to speed things up a little. Here are a few suggestions for your consideration.
-Julien

### CODE REVIEW

**Comment by jwoillez** [comment]
Note: The Travis failure is for python 3.3 and related to the build infrastructure, not the commits.

**Comment by romanz** [comment]
Thanks a lot for the contribution!
Will review the PR tomorrow :)

File: amodem/main.py [hunk 1]
```diff
@@ -59,7 +59,8 @@ def recv(config, src, dst, dump_audio=None, pylab=None):
         gain = 1.0 / amplitude
         log.debug('Gain correction: %.3f', gain)
 
-        sampler = sampling.Sampler(signal, sampling.Interpolator(), freq=freq)
+        sampler = sampling.Sampler(signal, sampling.defaultInterpolator,
```
Comment (romanz) [inline amodem/main.py]: I'm not sure why this change speeds up `recv()`.

Comment (romanz) [reply amodem/main.py]: Isn't the `sampling.Interpolator()` object is created once and used once (in both cases)?
Maybe I'm missing something...

**Comment by jwoillez** [comment]
Let me know if you need a rebase after c3cefee.

**Comment by romanz** [comment]
Thanks again :)

**Comment by jwoillez** [comment]
Thank -you-!
